main.gd

```diff
@@ -190,15 +203,17 @@ func _reset():
 func _call_gj_api(type, parameters):
 	if busy:
 		requestError = ERR_BUSY
-		emit_signal('gamejolt_request_completed')
+#		emit_signal('gamejolt_request_completed')
 		return
 	busy = true
 	_reset()
 	var url = _compose_url(type, parameters)
+	lasttype=type
 	requestError = request(url)
 	if requestError != OK:
-		busy = false
```

Comment:
No. The concept is - request is completed with request Error. This is why, the signal is also emitted here.